src/Transfer/Examples/YamlToEzPlatformContentType/resources/yaml/mini.yml

```diff
@@ -0,0 +1,6 @@
+banner:
+    fields:
+        title:
+            type: ezstring
+        description:
+            type: ezstring
```

Comment:
@valisj The (yaml to array and..) Array to ContentTypeObject does not really need the type defined when its ezstring, as it's the fallback value, but I'm having trouble telling yaml that the title and descrition should be a container/array when it does not contain any values (in this case type: ezstring). 

So if I knew how to write yaml properly this only needs to be those 4 lines, intead of 6. It's not a biggie, but would be nice I think.